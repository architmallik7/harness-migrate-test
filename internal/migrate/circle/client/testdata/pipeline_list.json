{
    "next_page_token": null,
    "items": [
        {
            "id": "eb02dad5-96d9-478f-aeea-2db0749d405e",
            "errors": [],
            "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
            "updated_at": "2023-01-30T17:12:27.243Z",
            "number": 5,
            "trigger_parameters": {
                "gitlab": {
                    "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
                    "x_gitlab_event_id": "",
                    "commit_author_name": "Ash Williams",
                    "user_id": "13246192",
                    "user_name": "Ash Williams",
                    "user_username": "ashwilliams1",
                    "branch": "main",
                    "commit_title": "Update config.yml",
                    "tag": "",
                    "commit_message": "Update config.yml",
                    "total_commits_count": "1",
                    "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
                    "user_avatar": "https://secure.gravatar.com/avatar/87d90fb1433c3f69adb0bf2acaee7de9?s=80&d=identicon",
                    "type": "push",
                    "project_id": "41770570",
                    "ref": "refs/heads/main",
                    "repo_name": "hello-gitlab",
                    "commit_author_email": "ash.williams@myyahoo.com",
                    "checkout_sha": "fca44e36c87ade8ab0c394044c7f95dc5fe88e0d",
                    "commit_timestamp": "2023-01-30T17:12:25+00:00",
                    "default_branch": "",
                    "commit_sha": "fca44e36c87ade8ab0c394044c7f95dc5fe88e0d",
                    "is_fork_merge_request": false
                },
                "circleci": {
                    "config_source_id": "d39e8f59-654b-4547-85c9-72b84c80d0ca",
                    "event_time": "2023-01-30T17:12:26.658Z",
                    "actor_id": "26d1ed6f-2a6c-4664-a44d-8ca444adae30",
                    "trigger_id": "95db7f52-118b-40dd-b694-833cb38f77db",
                    "event_type": "push",
                    "trigger_type": "gitlab"
                },
                "git": {
                    "tag": "",
                    "checkout_sha": "fca44e36c87ade8ab0c394044c7f95dc5fe88e0d",
                    "ref": "refs/heads/main",
                    "branch": "main",
                    "checkout_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git"
                }
            },
            "state": "created",
            "created_at": "2023-01-30T17:12:27.243Z",
            "trigger": {
                "received_at": "2023-01-30T17:12:26.658Z",
                "type": "gitlab",
                "actor": {
                    "login": "",
                    "avatar_url": null
                }
            }
        },
        {
            "id": "1af87b45-02e9-467e-be19-e4ae74fc114e",
            "errors": [],
            "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
            "updated_at": "2023-01-19T00:01:30.849Z",
            "number": 4,
            "trigger_parameters": {
                "gitlab": {
                    "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
                    "x_gitlab_event_id": "",
                    "commit_author_name": "Ash Williams",
                    "user_id": "13246192",
                    "user_name": "Ash Williams",
                    "user_username": "ashwilliams1",
                    "branch": "main",
                    "commit_title": "Update config.yml",
                    "tag": "",
                    "commit_message": "Update config.yml",
                    "total_commits_count": "1",
                    "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
                    "user_avatar": "https://secure.gravatar.com/avatar/87d90fb1433c3f69adb0bf2acaee7de9?s=80&d=identicon",
                    "type": "push",
                    "project_id": "41770570",
                    "ref": "refs/heads/main",
                    "repo_name": "hello-gitlab",
                    "commit_author_email": "ash.williams@myyahoo.com",
                    "checkout_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "commit_timestamp": "2023-01-19T00:01:28+00:00",
                    "default_branch": "",
                    "commit_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "is_fork_merge_request": false
                },
                "circleci": {
                    "config_source_id": "d39e8f59-654b-4547-85c9-72b84c80d0ca",
                    "event_time": "2023-01-19T00:01:30.356Z",
                    "actor_id": "26d1ed6f-2a6c-4664-a44d-8ca444adae30",
                    "trigger_id": "95db7f52-118b-40dd-b694-833cb38f77db",
                    "event_type": "push",
                    "trigger_type": "gitlab"
                },
                "git": {
                    "tag": "",
                    "checkout_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "ref": "refs/heads/main",
                    "branch": "main",
                    "checkout_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git"
                }
            },
            "state": "created",
            "created_at": "2023-01-19T00:01:30.849Z",
            "trigger": {
                "received_at": "2023-01-19T00:01:30.356Z",
                "type": "gitlab",
                "actor": {
                    "login": "",
                    "avatar_url": null
                }
            }
        },
        {
            "id": "dca5a820-e197-49ee-95b1-7bbb8dae4062",
            "errors": [],
            "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
            "updated_at": "2023-01-19T00:01:30.848Z",
            "number": 3,
            "trigger_parameters": {
                "gitlab": {
                    "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
                    "x_gitlab_event_id": "",
                    "commit_author_name": "Ash Williams",
                    "user_id": "13246192",
                    "user_name": "Ash Williams",
                    "user_username": "ashwilliams1",
                    "branch": "main",
                    "commit_title": "Update config.yml",
                    "tag": "",
                    "commit_message": "Update config.yml",
                    "total_commits_count": "1",
                    "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
                    "user_avatar": "https://secure.gravatar.com/avatar/87d90fb1433c3f69adb0bf2acaee7de9?s=80&d=identicon",
                    "type": "push",
                    "project_id": "41770570",
                    "ref": "refs/heads/main",
                    "repo_name": "hello-gitlab",
                    "commit_author_email": "ash.williams@myyahoo.com",
                    "checkout_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "commit_timestamp": "2023-01-19T00:01:28+00:00",
                    "default_branch": "",
                    "commit_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "is_fork_merge_request": false
                },
                "circleci": {
                    "config_source_id": "d39e8f59-654b-4547-85c9-72b84c80d0ca",
                    "event_time": "2023-01-19T00:01:30.344Z",
                    "actor_id": "26d1ed6f-2a6c-4664-a44d-8ca444adae30",
                    "trigger_id": "a3e5b590-261b-4186-803f-edb342daee4b",
                    "event_type": "push",
                    "trigger_type": "gitlab"
                },
                "git": {
                    "tag": "",
                    "checkout_sha": "eeb0ea8981749c1f0f7492794f9f67fc0a9c7b47",
                    "ref": "refs/heads/main",
                    "branch": "main",
                    "checkout_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git"
                }
            },
            "state": "created",
            "created_at": "2023-01-19T00:01:30.848Z",
            "trigger": {
                "received_at": "2023-01-19T00:01:30.344Z",
                "type": "gitlab",
                "actor": {
                    "login": "",
                    "avatar_url": null
                }
            }
        },
        {
            "id": "13280f67-cfd4-4ef0-95cc-8b1d95d1fae6",
            "errors": [],
            "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
            "updated_at": "2023-01-18T23:56:31.364Z",
            "number": 2,
            "trigger_parameters": {
                "gitlab": {
                    "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
                    "x_gitlab_event_id": "",
                    "commit_author_name": "Ash Williams",
                    "user_id": "13246192",
                    "user_name": "Ash Williams",
                    "user_username": "ashwilliams1",
                    "branch": "main",
                    "commit_title": "Minor yaml change",
                    "tag": "",
                    "commit_message": "Minor yaml change",
                    "total_commits_count": "1",
                    "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
                    "user_avatar": "https://secure.gravatar.com/avatar/87d90fb1433c3f69adb0bf2acaee7de9?s=80&d=identicon",
                    "type": "push",
                    "project_id": "41770570",
                    "ref": "refs/heads/main",
                    "repo_name": "hello-gitlab",
                    "commit_author_email": "ash.williams@myyahoo.com",
                    "checkout_sha": "aab7d9ea266fe1c25fb08c3b7411bc651fc9d0a8",
                    "commit_timestamp": "2023-01-18T23:56:29+00:00",
                    "default_branch": "",
                    "commit_sha": "aab7d9ea266fe1c25fb08c3b7411bc651fc9d0a8",
                    "is_fork_merge_request": false
                },
                "circleci": {
                    "config_source_id": "d39e8f59-654b-4547-85c9-72b84c80d0ca",
                    "event_time": "2023-01-18T23:56:30.925Z",
                    "actor_id": "26d1ed6f-2a6c-4664-a44d-8ca444adae30",
                    "trigger_id": "95db7f52-118b-40dd-b694-833cb38f77db",
                    "event_type": "push",
                    "trigger_type": "gitlab"
                },
                "git": {
                    "tag": "",
                    "checkout_sha": "aab7d9ea266fe1c25fb08c3b7411bc651fc9d0a8",
                    "ref": "refs/heads/main",
                    "branch": "main",
                    "checkout_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git"
                }
            },
            "state": "created",
            "created_at": "2023-01-18T23:56:31.364Z",
            "trigger": {
                "received_at": "2023-01-18T23:56:30.925Z",
                "type": "gitlab",
                "actor": {
                    "login": "",
                    "avatar_url": null
                }
            }
        },
        {
            "id": "0f888f7e-aaff-444b-9199-0ef458a5c370",
            "errors": [],
            "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
            "updated_at": "2023-01-18T20:00:30.384Z",
            "number": 1,
            "trigger_parameters": {
                "gitlab": {
                    "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
                    "x_gitlab_event_id": "",
                    "commit_author_name": "Ash Williams",
                    "user_id": "13246192",
                    "user_name": "Ash Williams",
                    "user_username": "ashwilliams1",
                    "branch": "main",
                    "commit_title": "Add new file",
                    "tag": "",
                    "commit_message": "Add new file",
                    "total_commits_count": "1",
                    "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
                    "user_avatar": "https://secure.gravatar.com/avatar/87d90fb1433c3f69adb0bf2acaee7de9?s=80&d=identicon",
                    "type": "push",
                    "project_id": "41770570",
                    "ref": "refs/heads/main",
                    "repo_name": "hello-gitlab",
                    "commit_author_email": "ash.williams@myyahoo.com",
                    "checkout_sha": "9360489dcaa8a0e57523fc1504f6edeea1ed7a51",
                    "commit_timestamp": "2023-01-18T20:00:28+00:00",
                    "default_branch": "",
                    "commit_sha": "9360489dcaa8a0e57523fc1504f6edeea1ed7a51",
                    "is_fork_merge_request": false
                },
                "circleci": {
                    "config_source_id": "d39e8f59-654b-4547-85c9-72b84c80d0ca",
                    "event_time": "2023-01-18T20:00:29.879Z",
                    "actor_id": "26d1ed6f-2a6c-4664-a44d-8ca444adae30",
                    "trigger_id": "95db7f52-118b-40dd-b694-833cb38f77db",
                    "event_type": "push",
                    "trigger_type": "gitlab"
                },
                "git": {
                    "tag": "",
                    "checkout_sha": "9360489dcaa8a0e57523fc1504f6edeea1ed7a51",
                    "ref": "refs/heads/main",
                    "branch": "main",
                    "checkout_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git"
                }
            },
            "state": "created",
            "created_at": "2023-01-18T20:00:30.384Z",
            "trigger": {
                "received_at": "2023-01-18T20:00:29.879Z",
                "type": "gitlab",
                "actor": {
                    "login": "",
                    "avatar_url": null
                }
            }
        }
    ]
}