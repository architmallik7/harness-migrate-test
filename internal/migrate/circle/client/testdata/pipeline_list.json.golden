[
  {
    "id": "eb02dad5-96d9-478f-aeea-2db0749d405e",
    "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
    "number": 5,
    "state": "created",
    "created_at": "2023-01-30T17:12:27.243Z",
    "updated_at": "2023-01-30T17:12:27.243Z",
    "trigger_parameters": {
      "gitlab": {
        "project_id": "41770570",
        "repo_name": "hello-gitlab",
        "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
        "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
        "branch": "main",
        "default_branch": ""
      }
    }
  },
  {
    "id": "1af87b45-02e9-467e-be19-e4ae74fc114e",
    "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
    "number": 4,
    "state": "created",
    "created_at": "2023-01-19T00:01:30.849Z",
    "updated_at": "2023-01-19T00:01:30.849Z",
    "trigger_parameters": {
      "gitlab": {
        "project_id": "41770570",
        "repo_name": "hello-gitlab",
        "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
        "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
        "branch": "main",
        "default_branch": ""
      }
    }
  },
  {
    "id": "dca5a820-e197-49ee-95b1-7bbb8dae4062",
    "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
    "number": 3,
    "state": "created",
    "created_at": "2023-01-19T00:01:30.848Z",
    "updated_at": "2023-01-19T00:01:30.848Z",
    "trigger_parameters": {
      "gitlab": {
        "project_id": "41770570",
        "repo_name": "hello-gitlab",
        "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
        "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
        "branch": "main",
        "default_branch": ""
      }
    }
  },
  {
    "id": "13280f67-cfd4-4ef0-95cc-8b1d95d1fae6",
    "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
    "number": 2,
    "state": "created",
    "created_at": "2023-01-18T23:56:31.364Z",
    "updated_at": "2023-01-18T23:56:31.364Z",
    "trigger_parameters": {
      "gitlab": {
        "project_id": "41770570",
        "repo_name": "hello-gitlab",
        "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
        "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
        "branch": "main",
        "default_branch": ""
      }
    }
  },
  {
    "id": "0f888f7e-aaff-444b-9199-0ef458a5c370",
    "project_slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS/TaCb55wNToTQJA9NpwJ368",
    "number": 1,
    "state": "created",
    "created_at": "2023-01-18T20:00:30.384Z",
    "updated_at": "2023-01-18T20:00:30.384Z",
    "trigger_parameters": {
      "gitlab": {
        "project_id": "41770570",
        "repo_name": "hello-gitlab",
        "repo_url": "git@gitlab.com:ashwilliams1/hello-gitlab.git",
        "web_url": "https://gitlab.com/ashwilliams1/hello-gitlab",
        "branch": "main",
        "default_branch": ""
      }
    }
  }
]